src/warc2zim/content_rewriting/css.py

```diff
@@ -23,14 +36,26 @@ def rewrite(self, content: str | bytes) -> str:
             rules = parse_stylesheet(content)
         self.process_list(rules)
 
-        output = serialize(rules)
+        try:
```

Comment:
Why are we catching exception only in serialization and not the whole processing? At least it is not homogeneous with the code for inline CSS processing, which makes it weird and at least deserve a comment explaining the reason.